### PR TITLE
Web Cert Course Number Overrides

### PR DESCRIPTION
**Pull request by mhoeber**
Add info about how to override a course number on web certificates.

Date Needed: 10/21/15
- [ ] Subject matter expert: @mattdrayer 
- [ ] Subject matter expert: 
- [x] Doc team review (level): @lamagnifica 
- [ ] Product review:
- [ ] Partner support:
- [ ] PM review: 
- [x] Ran en_us/make without warnings or errors
### Post-review
- [ ] Add description to release notes task as a comment
- [x] Update change log
- [ ] Squash commits

### CODE REVIEW

File: en_us/course_authors/source/front_matter/change_log.rst [hunk 1]
```diff
@@ -22,6 +22,9 @@ October 2015
 
    * - Date
      - Change
+   * - 20 October 2015
+     - Added the section :ref:`Set a Course Number Override` to :ref:`Setting
```
Comment (lamagnifica) [inline en_us/course_authors/source/front_matter/change_log.rst]: try to choose just one link to send people to, per a11y discussions.

File: en_us/open_edx_course_authors/source/front_matter/change_log.rst [hunk 2]
```diff
@@ -12,6 +12,9 @@ October 2015
 
    * - Date
      - Change
+   * - 20 October 2015
+     - Added the section :ref:`Set a Course Number Override` to :ref:`Setting
+       Up Course Certificates`.
```
Comment (lamagnifica) [inline en_us/open_edx_course_authors/source/front_matter/change_log.rst]: same

File: en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst [hunk 3]
```diff
@@ -85,9 +110,14 @@ To create a certificate for your course, follow these steps.
    new certificate**.
 
 #. Optionally, specify an alternative course title to use on the certificate
-   instead of the official course title. You might want to use a different
-   title on your course certificates if, for example, the official course
-   name is too long to fit well on the certificate.
+   in the **Course Title Override** field. 
+
+   You might want to use a different title on your course certificates if, for
+   example, the official course name is too long to fit well on the
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst]: maybe drop "well"?

File: en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst [hunk 4]
```diff
@@ -85,9 +110,14 @@ To create a certificate for your course, follow these steps.
...
+   certificate.
+
+   To specify an alternative course number, see `Set an Course Number
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst]: an --> a (in xref)

File: en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst [hunk 5]
```diff
@@ -68,6 +68,31 @@ course.
...
+******************************
+Set a Course Number Override
+******************************
+
+Optionally, you can specify an alternative course number to use on the
+certificate.
+
+You might want to use a different number on your course certificates if, for
+example, the official course number is meaningful only within your institution.
+
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst]: add a parallel xref here to the topic on overriding the name?

File: en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst [hunk 6]
```diff
@@ -68,6 +68,31 @@ course.
 
 #. At the bottom of the page, select **Save Changes**.
 
+.. _Set a Course Number Override:
+
+******************************
+Set a Course Number Override
+******************************
+
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst]: does this have to be done before you create the certificates? this seems like a somewhat odd place for this topic if it is optional. If you do have to decide and complete this before the "create" stage, maybe state that?

**Comment by lamagnifica** [comment]
some suggestions. :+1:

File: en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst (hunk 6 above)
Comment (mhoeber) [inline en_us/shared/building_and_running_chapters/building_course/creating_course_certificates.rst]: no, doesn't have to be before creating; change will get reflected after creation too. I'll move to after create/edit tasks

**Comment by mhoeber** [comment]
Done, merge when 10/21 release is out

**Comment by mhoeber** [comment]
jenkins test this

**Pull request merged**